pkg/crc/machine/bundle/metadata.go

```diff
@@ -353,7 +353,12 @@ func Download(preset crcPreset.Preset, bundleURI string) (string, error) {
 	// bundles, their sha256sums are known and can be checked.
 	if bundleURI == constants.GetDefaultBundlePath(preset) {
 		if preset == crcPreset.OpenShift || preset == crcPreset.Microshift {
-			return DownloadDefault(preset)
+			downloadedBundlePath, err := DownloadDefault(preset)
+			if err != nil {
```

Comment:
This cannot be automatic and unconditional.

Reply:
To expand on this, this can go behind a config option, or maybe env var, or ... ?
I'd also add a log message to inform the user.